content/en/tests/setup/ruby.md

```diff
@@ -163,13 +191,6 @@ The Cucumber integration traces executions of scenarios and steps when using the
 
 To activate your integration, add the following code to your application:
 
-<!-- TODO: Explicitly setting `c.tracing.enabled` overrides any existing value, including the environment
```

Comment:
this comment is no longer relevant as we no longer recommend changing `c.tracing.enabled` for CI config